src/gigs-board/components/organism/editor.jsx

```diff
@@ -217,6 +217,11 @@ const fieldParamsByType = {
     name: "components.molecule.text-input",
     inputProps: { type: "text" },
   },
+
+  markdown: {
+    name: "components.molecule.markdown-editor",
+    className: "px-2"
```

Comment:
Let's remove this class and pass it only in particular cases when it's really needed